wine-classification/{{cookiecutter.project_name}}/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flytekit==1.4.2
```

Comment:
I'd get latest flytekit, >= 1.4.2 for instance. There might be some bug fixes we need to ship and backport.

Reply:
OK